README.md

```diff
@@ -29,7 +29,8 @@ Usage: Erlang LS [-v] [-t [<transport>]] [-p [<port>]] [-d [<log_dir>]]
 
   -v, --version    Print the current version of Erlang LS
   -t, --transport  Specifies the transport the server will use for the
-                   connection with the client. [default: tcp]
+                   connection with the client, either "tcp" or "stdio".
```

Comment:
Please add this to the options returned by `--help` [here](https://github.com/erlang-ls/erlang_ls/blob/master/src/erlang_ls.erl#L68).